src/overlay_params.cpp

```diff
@@ -835,7 +835,6 @@ parse_overlay_config(struct overlay_params *params,
    else
       HUDElements.text_column = 1;
 
-   if(logger && logger->m_params == nullptr) logger.reset();
```

Comment:
Why did you decide to remove this line?
Without this the logger will never reinitialize and we only save the param values when the logger initializes.
So if you were to change `output_path` while running mangohud, it wouldn't actually change in the logger